ci/release/update-version.sh

```diff
@@ -30,5 +32,5 @@ function sed_runner() {
     sed -i.bak ''"$1"'' $2 && rm -f ${2}.bak
 }
 
-
-# No-op
\ No newline at end of file
+# Update UCX-Py version
+sed_runner 's/export UCXPY_VERSION=.*/export UCXPY_VERSION='${NEXT_UCXPY_VERSION}'/g' ci/gpu/build.sh
```

Comment:
I think we should use `"` instead of `'` to expand the variable

Reply:
No objections from me, I had copied the format from cuDF's script, but I now see it mixes both formats. For example https://github.com/rapidsai/cudf/blob/67c925c84aa111ad3a54c0352c18c570d17f5d75/ci/release/update-version.sh#L34 and https://github.com/rapidsai/cudf/blob/67c925c84aa111ad3a54c0352c18c570d17f5d75/ci/release/update-version.sh#L58